src/components/templates/Embeddable/Embeddable.scss

```diff
@@ -13,6 +16,13 @@
     width: 90%;
 
     border: none;
-    background-color: $light-grey;
+    z-index: z(embeddable-iframe);
+  }
+
+  &__background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2